ext/FullScreenTriangle/FullScreenTriangle.h

```diff
@@ -28,8 +28,9 @@ void main()
     TexCoord = tc[gl_VertexIndex];
 }
 	)===";
-	auto shader = driver->createGPUShader(core::smart_refctd_ptr<ICPUShader>(source));
-	std::get<0>(retval) = driver->createGPUSpecializedShader(shader.get(), nullptr);
+	auto shader = driver->createGPUShader(core::make_smart_refctd_ptr<asset::ICPUShader>(source));
+    video::IGPUSpecializedShader::SInfo specInfo({}, nullptr, "main", video::IGPUSpecializedShader::ESS_VERTEX);
+	std::get<0>(retval) = driver->createGPUSpecializedShader(shader.get(), std::move(specInfo));
```

Comment:
maybe we should refactor this extension and have an `init` function that adds this shader to the built-ins?
(or just put this vertex shader in built-in engine core)

Reply:
?

Reply:
?

Reply:
IMO would seem a little weird that shader is builtin in the cache and other components of retval are not

Reply:
ok lets leave it, FS triangles are dumb to use anyway (now that we have COMPUTE)